src/applications/personalization/profile/components/ProfileInfoCard.jsx

```diff
@@ -153,33 +140,38 @@ List.propTypes = {
 const Sections = ({ data, level }) => {
   return (
     <>
-      {data.map((rowData, index) => (
-        <div
-          key={index}
-          className={index === 0 ? classes.firstRow : classes.secondaryRow}
-          id={rowData.id}
-        >
-          {rowData.title && (
-            <>
-              <HeadingLevel
-                className={classes.rowTitle}
-                level={level}
-                hasRowDescription={!!rowData?.description}
-              >
-                {rowData.title}
-                {rowData.alertMessage && <>{rowData.alertMessage}</>}
-              </HeadingLevel>
-              {rowData.description && (
-                <RowDescription description={rowData.description} />
-              )}
-            </>
-          )}
-
-          {rowData?.value && (
-            <div className={classes.rowValue}>{rowData.value}</div>
-          )}
-        </div>
-      ))}
+      {data.map((rowData, index) => {
+        // heading should only have bottom margin when there is no description
+        const rowHeadingClasses = classNames([
+          classes.rowTitle,
+          {
+            'vads-u-margin-bottom--1': !rowData?.description,
+          },
+        ]);
```

Comment:
Moved the conditional logic to here so that it is only adding this class for a card.rowTitle if accompanied by a description